clippy_lints/src/methods.rs

```diff
@@ -653,7 +670,8 @@ impl LintPass for Pass {
             GET_UNWRAP,
             STRING_EXTEND_CHARS,
             ITER_CLONED_COLLECT,
-            USELESS_ASREF
+            USELESS_ASREF,
+            FOLD_ANY
```

Comment:
Nit: Does this macro support trailing commas? If so, please add one and the next diff is only one line :)

Reply:
It appears not:

```
error: unexpected end of macro invocation
   --> clippy_lints/src/methods.rs:674:21
    |
674 |             FOLD_ANY,
    |                     ^
```

Reply:
I've created an issue https://github.com/rust-lang-nursery/rust-clippy/issues/2352. I'll do this next weekend if it's still open.

Reply:
Oh, this is a lint defined in the compiler, so that issue is probably more useful over at <https://github.com/rust-lang/rust> :)

Reply:
Nuts. https://github.com/rust-lang/rust/issues/47428